### DISCUSSION TITLE
[Bug]: Waline无法适配黑暗模式

### DISCUSSION THREAD

**Discussion by MrZhang365** [walinejs/waline#1715] [Q&A] [answered]
### 问题描述 | Describe the bug

https://i.postimg.cc/pVw7wJMJ/image.png  
如图所示，我在Hexo的NexT主题上启动了黑暗模式，但是Waline评论区的字体颜色却是深色的，导致看不清文字。

### 问题网站 | Website URL

https://小张.就是傻逼.cf

### 服务部署在哪里？ | Where your waline deploy?

Vercel (Default)

### 数据存储在哪里？| Where your comment data store?

LeanCloud(https://leancloud.app)

**Comment by Mister-Hope** [comment, marked as answer]
https://waline.js.org/guide/features/style.html#%E6%9A%97%E9%BB%91%E6%A8%A1%E5%BC%8F%E6%94%AF%E6%8C%81
